player/tests.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+from .models import Player, BallGame
+import pytest
+
+
+@pytest.mark.django_db
+class TestPlayerModel:
+
+    @pytest.fixture
+    def user(self):
+        return get_user_model().objects.create_user(
+            username='testuser',
+            password='testpass'
+        )
+
+    @pytest.fixture
+    def saved_player(self, user):
+        player = Player.objects.create(
+            user=user,
+            birth_date='1990-01-01',
+            favorite_ball_game=BallGame.Soccer
+        )
+        return player
+
+    def test_create_player(self, saved_player, user):
+        player = saved_player
+        assert player.user == user
+        assert player.birth_date == '1990-01-01'
+        assert player.favorite_ball_game == BallGame.Soccer
+
+    def test_update_player(self, saved_player):
+        player = saved_player
+        player.favorite_ball_game = BallGame.Basketball
+        player.save()
+        updated_player = Player.objects.get(pk=player.pk)
+        assert updated_player.favorite_ball_game == BallGame.Basketball
```

Comment:
You need this sort of assertion when creating a player as well.

Reply:
more precisely. You need to verify that you can pull the player from the db and that it's the same player as you were expecting to find.

Reply:
added this assertion: 
        assert updated_player.pk == saved_player.pk
how's that ?

> more precisely, you need to verify that you can pull the player from the db and that it's the same player as you were expecting to find.

I am pulling the saved player by it's id so how could i get other player ?

Reply:
This comment was not about this test, but about the test in which you create a player. There, you need to verify that you're able to fetch the instance.